test/test_decimal128.cpp

```diff
@@ -0,0 +1,113 @@
+/*************************************************************************
+ *
+ * Copyright 2019 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **************************************************************************/
+
+#include <realm.hpp>
+#include <realm/array_decimal128.hpp>
+
+#include "test.hpp"
+
+
+using namespace realm;
+
```

Comment:
It might be a good idea to have a wrapper test here which triggers the provided external Intel tests.

Reply:
Maybe at some point. But it should not be done as part of the normal test execution. We are not planning to modify the Intel library.

Reply:
Good point. Maybe the more useful thing would be to import a bunch of MongoDB's Decimal128 tests to make sure we are compliant with their implementation.